jdbc-driver/src/test/java/com/google/cloud/dataproc/jdbc/DataprocDriverTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Google Inc.
```

Comment:
Copyrights should be to `Google LLC`, not `Google Inc` - this changed couple years ago:

```
Copyright 2020 Google LLC

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    https://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
```

Reply:
Should I add copyrights to Gradle build files as well? I've seen other repos with copyrights in pom.xml.

Reply:
Yes, you should copyright Gradle configs that you wrote, but not Gradle Wrapper configs that were auto-generated by Gradle.

Reply:
I see, thanks.